tests/test_service.py

```diff
@@ -288,3 +288,30 @@ def test_positive_foreman_maintain_service_list_sidekiq(ansible_module):
         assert "dynflow-sidekiq@orchestrator" in result["stdout"]
         assert "dynflow-sidekiq@worker" in result["stdout"]
         assert "dynflow-sidekiq@worker-hosts-queue" in result["stdout"]
+
+
+def test_positive_foreman_maintain_service_status_bz1945916(ansible_module, setup_rpmsave_file):
+    """Verify foreman-maintain service status does not get failed because of .rpmsave files
+
+    :id: dda696c9-7385-4450-8380-b694e4016661
+
+    :setup:
+        1. foreman-maintain should be installed.
+
+    :steps:
+        1. Run foreman-maintain service status.
+        2. Verify foreman-maintain service status does not get failed because of .rpmsave files
+
+
+    :expectedresults: foreman-maintain service status should be passed
+
+    :BZ: 1945916
```

Comment:
```suggestion
    :BZ: 1945916, 1962853
```
Also, need to cherrypick for the 6.9.z branch.